spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -1027,12 +1022,13 @@ def register_wrapper(fname_src, fname_dest, param, paramregmulti, fname_src_seg=
         if (not same_space and i_step > 0) or (same_space and i_step > 1):
             printv('\nApply transformation from previous step', param.verbose)
             for ifile in range(len(src)):
-                sct_apply_transfo.main(args=[
+                sct_apply_transfo.main(argv=[
                     '-i', src[ifile],
                     '-d', dest[ifile],
-                    '-w', warp_forward,
                     '-o', add_suffix(src[ifile], '_reg'),
-                    '-x', interp_step[ifile]])
+                    '-x', interp_step[ifile],
+                    '-w'] + warp_forward
```

Comment:
Replacing `['-w', warp_forward]` with `['-w'] + warp_forward` to deal with nested list.

(This was the only call to `sct_apply_transfo.main()` that used nested lists.)